.github/workflows/markdown-link-checker.yaml

```diff
@@ -0,0 +1,15 @@
+name: 🏀 Markdown Link Check
+
+on: [pull_request]
+
+jobs:
+  markdown-link-check:
+    name: 🧹 Check Links
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: gaurav-nelson/github-action-markdown-link-check@v1
```

Comment:
This GitHub Action needs to be removed since it is a third party Action. There are more security risks using third party actions.

Official -> "The individual jobs in a workflow can interact with (and compromise) other jobs. For example, a job querying the environment variables used by a later job, writing files to a shared directory that a later job processes, or even more directly by interacting with the Docker socket and inspecting other running containers and executing commands in them.

This means that a compromise of a single action within a workflow can be very significant, as that compromised action would have access to all secrets configured on your repository, and can use the GITHUB_TOKEN to write to the repository. Consequently, there is significant risk in sourcing actions from third-party repositories on GitHub."

I will need to use the official Node Action and install `markdown-link-check` manually with npm 

https://docs.github.com/en/actions/learn-github-actions/security-hardening-for-github-actions#using-third-party-actions
https://github.com/marketplace/actions/markdown-link-check
https://github.com/marketplace/actions/setup-node-js-environment
https://github.com/tcort/markdown-link-check